### PR TITLE
resend verification link bug fix

### LINKED ISSUE #514: bug in resend verification link.
parameter name in request is 'emailid'
parameter extracted from the request is 'emailId'
this is a typo which leads to null pointer exception

### PR DESCRIPTION
**Pull request by DravitLochan**
Fixes #514

### CODE REVIEW

**Comment by DravitLochan** [comment]
@mariobehling Please merge this

File: src/ai/susi/server/api/aaa/ResendVerificationLinkService.java [hunk 1]
```diff
@@ -32,12 +32,13 @@
 /**
  * Created by dravit on 25/7/17.
  * parameter : emailId
- * sample request : 
+ * sample request :
  * http://127.0.0.1:4000/aaa/resendVerificationLink.json?emailId=dravit.lochan@gmail.com
```
Comment (mariobehling) [inline src/ai/susi/server/api/aaa/ResendVerificationLinkService.java]: Why is your email in here?

Comment (mariobehling) [inline src/ai/susi/server/api/aaa/ResendVerificationLinkService.java]: Email IDs should be set in a global configuration. Please change this accordingly.

Comment (DravitLochan) [inline src/ai/susi/server/api/aaa/ResendVerificationLinkService.java]: @mariobehling This is just a sample request. In my earlier PR, I added example@example.com as  a sample API call but had to change it. Please direct me what email id should I put in sample requests.

**Comment by mariobehling** [comment]
Travis was failing. Please check again.

**Comment by DravitLochan** [comment]
@dynamitechetan has fixed in #529

**Comment by mariobehling** [comment]
What is the status on this now? Is this still needed?

**Comment by DravitLochan** [comment]
@mariobehling yes. This is still needed.

**Pull request merged**